### DISCUSSION TITLE
How to PR for individual folder/file ?

### DISCUSSION THREAD

**Discussion by soumyaranjan-panda** [Spectrum-CETB/Spectober_Fest#28] [Q&A] [answered]
How to PR for individual folder/file?
when I try to pr a folder every other folder gets included in it.

**Comment by yagyandatta** [comment, marked as answer]
you can't actually. You can send a pull request for a specific branch only.

**Reply by Aditya-150** [reply]
> you can't actually. You can send a pull request for a specific branch only.

@soumyaranjan-panda is your doubt clear. If it's not I can explain it further if you want. Just reply back to this.

**Reply by soumyaranjan-panda** [reply]
yes understood
